Assets/Global/Scripts/RoomLogic/RoomTransitionDoor.cs

```diff
@@ -16,18 +16,27 @@ public class RoomTransitionDoor : Interactable
     [SerializeField] private Animator animator;
     [SerializeField] private MeshRenderer doorMesh;
     [SerializeField] private string nextRoomName;
-    private bool isTransitioning = false;
+    public int nextRoomid;
+
     public CrossfadeController crossfadeController;
     public PlayerSpawnPoint playerSpawnPoint;
+    public DoorManager doorManager;
     
     private string currentScenename;
  
 
     private void Awake()
     {
+        IsDisabled = IsDisabled; // ugly fix so maybe we have to change in the future
```

Comment:
???? what is the meaning of this line code haha

Reply:
This is because the Setter of the `IsDisabled` property in the Interactable contains logic.
In the future we have to take this out of the setter and just create its own method for it. but for now this was a quick easy fix to active the sideeffect of the setter

![image](https://github.com/user-attachments/assets/bbec45ac-2703-47bc-91bb-bdb1320b1f9b)